build.gradle

```diff
@@ -51,7 +51,6 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
I think this was readded by accident during merge: 72fb395648d36480220869e99c271057b87b77d7

It was removed in 272ac6e82a08842d78a35c569a7bf5a5f3d8f6d0

Reply:
Oh yes, it was. Thank you!